### PR TITLE
[HOTFIX] Fixing UI Implementation

### PR DESCRIPTION
**Pull request by fxdros**
Several things have been done:
1. Move the `logout` button to the bottom of the sidebar
2. Add a default avatar icon on course menu
3. Change avatar circle size to 48px and change the bg-color to secondary
4. Give background color on the announcement to differentiate it from the comments section.
5. Edit people in the course list
6. Edit layout for `create topic`

Screen Shot 2022-06-10 at 06 29 05
Screen Shot 2022-06-10 at 06 29 11
Screen Shot 2022-06-10 at 06 29 17
Screen Shot 2022-06-10 at 06 29 21
Screen Shot 2022-06-10 at 06 35 50

### CODE REVIEW

File: src/resources/app/components/Container/AnnouncementContainer.vue [hunk 1]
```diff
@@ -37,6 +37,7 @@ export default {
       try {
         const response = await getAnnouncementsAPI(courseId);
         this.announcements = response.data;
+        console.log(this.announcements)
```
Comment (AldiNFitrah) [inline src/resources/app/components/Container/AnnouncementContainer.vue]: can it be removed?

File: src/resources/app/components/Sidebar.vue [hunk 2]
```diff
@@ -93,6 +93,7 @@
         </v-list-item-icon>
         <v-list-item-title>Settings</v-list-item-title>
       </v-list-item>
+      <div class="absolute bottom-7 w-11/12">
       <v-list-item link @click="logout" class="logout">
         <v-list-item-icon v-show="mini">
           <v-icon color="error">mdi-logout</v-icon>
```
Comment (AldiNFitrah) [inline src/resources/app/components/Sidebar.vue]: these should be intended I guess

File: src/resources/app/pages/CreateTopic.vue [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
+        </div>
+      </v-form>
+    </v-container>
+  </CreationLayout>
 </template>
 
 <script>
 import AppLayout from "../components/Layout/AppLayout.vue";
 import { createTopicAPI } from "../../api/topic";
+import CreationLayout from "../components/Layout/CreationLayout";
```
Comment (AldiNFitrah) [inline src/resources/app/pages/CreateTopic.vue]: minor: reorder the imports please

**Review by AldiNFitrah — COMMENTED** [review]
in the screenshots that you provided, the user avatar is still broken. Can we fix it?

**Review by AldiNFitrah — APPROVED** [review]

**Review by IlmaMannix — CHANGES_REQUESTED** [review]
thank uuuu bas ^^

**Review by IlmaMannix — APPROVED** [review]
thank u basss

**Pull request merged**